### PR TITLE
Collapsible

### LINKED ISSUE #48: Collapsible and diary
## User story

As a **user** I would like to have my diary entries displayed in a list and, if desired, have more details displayed so that I can better deal with my days, but still not lose the overview with lots of information.

## Notes / Links / Further Description
Check out collapsible tutorials

## ToDos

- [x]Create collapsible component
- [x]Add needed informations
- [x]Setup page with mockup data to map through
- [x]Create Storybook story
- [x]Change paths to Diary

### PR DESCRIPTION
**Pull request by slawoe**
Create a collapsible diary element for the diary page with all information given daily through the questionnaireform.

Fix #48

### CODE REVIEW

File: client/src/components/DiaryListElement.js [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+      </MainCardDiv>
+      <CollapsingDiv>
+        {open && (
+          <>
+            <DetailsInfoElement>
+              <span>WAS LIEF BESONDERS GUT:</span>
+              <span>{whatWasGood}</span>
+            </DetailsInfoElement>
+            <DetailsInfoElement>
+              <span>WAS HÄTTE BESSER LAUFEN KÖNNEN?:</span>
```
Comment (manuelsanchez2) [inline client/src/components/DiaryListElement.js:100]: I would remove the question mark here to keep the consistency with the other two spans

File: client/src/components/DiaryListElement.js [hunk 2]
```diff
@@ -0,0 +1,123 @@
+import React, { useState } from "react";
+import styled from "@emotion/styled";
+import PropTypes from "prop-types";
+
+const DiaryListItem = styled.div`
```
Comment (manuelsanchez2) [inline client/src/components/DiaryListElement.js:5]: I would add some -if not all- of these components apart so that this code is cleaner

**Review by manuelsanchez2 — APPROVED** [review]
Keep the good job!

File: client/src/components/DiaryListElement.js (hunk 1 above)
Comment (slawoe) [reply client/src/components/DiaryListElement.js:100]: Done. Thx!

File: client/src/components/DiaryListElement.js (hunk 2 above)
Comment (slawoe) [reply client/src/components/DiaryListElement.js:5]: You're right, but as I use these components just on one page and not reuse them - it makes no sense to make single components out of it.

**Pull request merged**